python/gui/locator/qgslocatorfilter.sip

```diff
@@ -173,6 +173,36 @@ class QgsLocatorFilter : QObject
  :rtype: bool
 %End
 
+    bool enabled() const;
+%Docstring
+ Returns true if the filter is enabled.
+.. seealso:: setEnabled()
+ :rtype: bool
+%End
+
+    void setEnabled( bool enabled );
+%Docstring
+ Sets whether the filter is ``enabled``.
+.. seealso:: enabled()
+%End
+
+    virtual bool hasConfigWidget() const;
+%Docstring
+ Should return true if the filter has a configuration widget.
+.. seealso:: createConfigWidget()
+ :rtype: bool
+%End
+
+    virtual void openConfigWidget( QWidget *parent = 0 );
```

Comment:
No - because that would transfer the filter to that parent. I think this is ok as is - it's up to the subclass implementation to manage the memory here.